PyInquirer/__init__.py

```diff
@@ -3,27 +3,14 @@
 from __future__ import absolute_import, print_function
 import os
 
-# from prompt_toolkit.token import Token
-# from prompt_toolkit.styles import style_from_dict
-# from prompt_toolkit.validation import Validator, ValidationError
```

Comment:
This was a mistake, but if only our tests rely on this the clean solution is to point those imports to prompt_toolkit